docs/api.md

```diff
@@ -5,71 +5,72 @@
-    - [[zoom]](#zoom)
-    - [[single-instance]](#single-instance)
-    - [[clear-cache]](#clear-cache)
-    - [[tray]](#tray)
-    - [[basic-auth-username]](#basic-auth-username)
-    - [[processEnvs]](#processenvs)
-    - [[file-download-options]](#file-download-options)
-    - [[always-on-top]](#always-on-top)
-    - [[global-shortcuts]](#global-shortcuts)
-    - [[browserwindow-options]](#browserwindow-options)
-    - [[darwin-dark-mode-support]](#darwin-dark-mode-support)
-    - [[background-color]](#background-color)
+  - [Target Url](#target-url)
+  - [[dest]](#dest)
+  - [Help](#help)
+  - [Version](#version)
+  - [[name]](#name)
+  - [[platform]](#platform)
+  - [[arch]](#arch)
+  - [[app-copyright]](#app-copyright)
+  - [[app-version]](#app-version)
+  - [[build-version]](#build-version)
+  - [[electron-version]](#electron-version)
+  - [[widevine]](#widevine)
+  - [[no-overwrite]](#no-overwrite)
+  - [[conceal]](#conceal)
+  - [[icon]](#icon)
+    - [Packaging for Windows](#packaging-for-windows)
+    - [Packaging for Linux](#packaging-for-linux)
+    - [Packaging for macOS](#packaging-for-macos)
+      - [Manually Converting `.icns`](#manually-converting-icns)
+  - [[counter]](#counter)
+  - [[bounce]](#bounce)
+  - [[width]](#width)
+  - [[height]](#height)
+  - [[min-width]](#min-width)
+  - [[min-height]](#min-height)
+  - [[max-width]](#max-width)
+  - [[max-height]](#max-height)
+  - [[x]](#x)
+  - [[y]](#y)
+  - [[show-menu-bar]](#show-menu-bar)
+  - [[fast-quit]](#fast-quit)
+  - [[user-agent]](#user-agent)
+  - [[honest]](#honest)
+  - [[ignore-certificate]](#ignore-certificate)
+  - [[disable-gpu]](#disable-gpu)
+  - [[ignore-gpu-blacklist]](#ignore-gpu-blacklist)
+  - [[enable-es3-apis]](#enable-es3-apis)
+  - [[insecure]](#insecure)
+  - [[internal-urls]](#internal-urls)
+  - [[block-external-urls]](#block-external-urls)
+  - [[proxy-rules]](#proxy-rules)
+  - [[flash]](#flash)
+  - [[flash-path]](#flash-path)
+  - [[disk-cache-size]](#disk-cache-size)
+  - [[inject]](#inject)
+  - [[full-screen]](#full-screen)
+  - [[maximize]](#maximize)
+  - [[hide-window-frame]](#hide-window-frame)
+  - [[title-bar-style]](#title-bar-style)
+  - [[verbose]](#verbose)
+  - [[disable-context-menu]](#disable-context-menu)
+  - [[disable-dev-tools]](#disable-dev-tools)
+  - [[crash-reporter]](#crash-reporter)
+  - [[zoom]](#zoom)
+  - [[single-instance]](#single-instance)
+  - [[clear-cache]](#clear-cache)
+  - [[tray]](#tray)
+  - [[basic-auth-username]](#basic-auth-username)
+  - [[processEnvs]](#processenvs)
+  - [[file-download-options]](#file-download-options)
+  - [[always-on-top]](#always-on-top)
+  - [[global-shortcuts]](#global-shortcuts)
+  - [[browserwindow-options]](#browserwindow-options)
+  - [[darwin-dark-mode-support]](#darwin-dark-mode-support)
+  - [[background-color]](#background-color)
+  - [[disable-old-nativefier-warning-yesiknowitisinsecure]](#disable-old-nativefier-warning-yesiknowitisinsecure)
```

Comment:
Naming nit again, here and elsewhere: this isn't about old nativefier, it's about old **build** / **Electron**. So, `--disable-old-electron-warning-yesiknowitisinsecure` or `--disable-old-build-warning-yesiknowitisinsecure`, as you prefer.